sdk/storage/azure-storage-files-shares/test/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
```

Comment:
Can this be dialed back to version 3.5?

Reply:
Not yet.
We would also need to break the Storage test generation into multiple exe files instead of just one in other to go back to previous CMake versions.
Currently, the main storage test exe is created in the main CMake file with:
`add_executable(azure-storage-test)` without adding any source files.
Then, later, each subdirectory adds source files to this exe target.  This is only supported on CMake version 3.13.

We can either list all the source files in the main CMake file (uggly solution), or let each subdirectory to create its own exe file.

That could be a different PR for the future as an effort to move CMake supported version back